File: 프로그래머스/0/181902. 문자 개수 세기/문자 개수 세기.py
def solution(my_string):
    answer = [0] * 52
    
    for i in range(len(my_string)):
        if my_string[i].isupper() == True:
            idx = ord(my_string[i]) - 65
        
        else:
            idx = ord(my_string[i]) - 71
        
        answer[idx] += 1
    
    return answer

File: 프로그래머스/0/181902. 문자 개수 세기/문자 개수 세기.js
function solution(my_string) {
    var answer = Array(52).fill(0);
    let idx = 0
    
    for(let s of my_string){
        if(isAllLowerCase(s))
            idx = s.charCodeAt(0)  - 71
        else
            idx = s.charCodeAt(0)  - 65
        answer[idx] += 1
    }
    
    return answer;
}

function isAllLowerCase(str) {
    return str === str.toLowerCase();
}
